client/src/components/GameInfoCarousel/styles.js

```diff
@@ -6,44 +6,55 @@ export const Container = styled.div`
   align-items: center;
   height: 100%;
   width: 100%;
-  background: url('https://store.cloudflare.steamstatic.com/public/images/v6/app/game_page_background_shadow.png?v=2');
-  background-position: bottom;
-  background-repeat: no-repeat;
   padding: 20px;
+  margin-top: 150px;
 `;
 
 export const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
-  width: 50%;
-  height: 50%;
+  align-items: center;
+`;
+
+export const BackgroundWaterMark = styled.div`
+  display: flex;
+  justify-content: center;
+  background: url('https://store.cloudflare.steamstatic.com/public/images/v6/app/game_page_background_shadow.png?v=2');
+  background-position: bottom;
+  background-repeat: no-repeat;
+  min-width: 885px;
+  max-width: 50%;
 `;
 
 export const MainGameInfoWrapper = styled.div`
   display: flex;
+  height: 50%;
+  width: 50%;
+  justify-content: center;
 
 ::before {
+    border-radius: 1px;
+    border-width: thick;
     content: "";
     background: url("${(props) => props.backgroundImage}");
```

Comment:
Super nit picky and not at all necessary, but you could probably use destructuring here to make it a bit cleaner:

background: url("${({ backgroundImage }) => backgroundImage}");